digital_image_processing/filters/laplacian_filter.py

```diff
@@ -0,0 +1,79 @@
+# @Author  : ojas-wani
+# @File    : laplacian_filter.py
+# @Date    : 10/04/2023
+
+import numpy as np
+from cv2 import (
+    BORDER_DEFAULT,
+    COLOR_BGR2GRAY,
+    CV_64F,
+    cvtColor,
+    filter2D,
+    imread,
+    imshow,
+    waitKey,
+)
+from digital_image_processing.filters.gaussian_filter import gaussian_filter
```

Comment:
I fixed this with `ruff --fix digital_image_processing/filters/laplacian_filter.py` which added one blank line between third-party imports and local imports.

```suggestion

from digital_image_processing.filters.gaussian_filter import gaussian_filter
```